crates/sui-framework/packages/sui-framework/sources/crypto/zklogin_verified_iss.move

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module sui::zklogin_verified_iss {
+    use std::string::String;
+    use sui::object;
+    use sui::object::UID;
+    use sui::tx_context::TxContext;
+    use sui::tx_context;
+
+    /// Error if the proof consisting of the inputs provided to the verification function is invalid.
+    const EInvalidInput: u64 = 0;
+
+    /// Error if the proof consisting of the inputs provided to the verification function is invalid.
+    const EInvalidProof: u64 = 1;
+
+    /// Posession of a VerifiedIssuer proves that the user's address was created using zklogin and with the given issuer
+    /// (identity provider).
+    struct VerifiedIssuer has key {
+
+        /// The ID of this VerifiedIssuer
+        id: UID,
+
+        /// The address this VerifiedID is associated with
+        address: address,
+
+        /// The issuer
+        iss: String,
+    }
+
+    /// Returns the address associated with the given VerifiedIssuer
+    public fun get_address(verified_issuer: &VerifiedIssuer): address {
+        verified_issuer.address
+    }
+
+    /// Returns the issuer associated with the given VerifiedIssuer
+    public fun get_iss(verified_issuer: &VerifiedIssuer): String {
+        verified_issuer.iss
+    }
```

Comment:
```suggestion
    public fun iss(verified_issuer: &VerifiedIssuer): &String {
        &verified_issuer.iss
    }
```
`borrow_iss` might also be canonical (though a bit strange if you do not have a non-borrowed function version)

Reply:
I would recommend doing this by-ref since the caller can always copy it if they need to